test/suites/integration/subnet_test.go

```diff
@@ -108,3 +134,43 @@ func getSubnets(tags map[string]string) map[string][]string {
 	Expect(err).To(BeNil())
 	return subnets
 }
+
+func enableResourceBasedNaming(subnetIDs ...string) {
+	for subnetID := range subnetIDs {
+		_, err := env.EC2API.ModifySubnetAttribute(&ec2.ModifySubnetAttributeInput{
+			EnableResourceNameDnsARecordOnLaunch: &ec2.AttributeBooleanValue{
+				Value: aws.Bool(true),
+			},
+			SubnetId: aws.String(subnetIDs[subnetID]),
+		})
+		Expect(err).To(BeNil())
+		_, err = env.EC2API.ModifySubnetAttribute(&ec2.ModifySubnetAttributeInput{
+			PrivateDnsHostnameTypeOnLaunch: aws.String("resource-name"),
+			SubnetId:                       aws.String(subnetIDs[subnetID]),
+		})
+		Expect(err).To(BeNil())
+	}
+}
+
+func disableResourceBasedNaming(subnetIDs ...string) {
```

Comment:
Can these be formatted as `ExpectResourceBasedNamingDisabled`? Generally, we are formatting things that have expect statements inside of them to start with `Expect...` to have visibility into the fact there is an Expect statement there at all

Reply:
Updated!